routes/api.js

```diff
@@ -447,25 +455,31 @@ router.get('/create_account', apiMiddleware(async (req) => {
     account_auths: accountAuths,
     key_auths: [[publicKeys.posting, 1]],
   };
-
-  try {
-    await steem.broadcast.accountCreateWithDelegationAsync(
-      process.env.DELEGATOR_ACTIVE_WIF,
-      process.env.CREATE_ACCOUNT_FEE,
-      process.env.CREATE_ACCOUNT_DELEGATION,
-      process.env.DELEGATOR_USERNAME,
-      username,
-      owner,
-      active,
-      posting,
-      publicKeys.memo,
-      metadata,
-      [],
-    );
-  } catch (cause) {
-    throw new ApiError({ type: 'error_api_create_account', cause });
+  const userWithHash = await req.db.users.findOne({ where: { email: decoded.email } });
```

Comment:
while it might be a rare edge case, we should handle it anyways because it's easy: mysql might return a cached result for this query.

this is how to circumvent the cache:
https://github.com/steemit/condenser/blob/master/src/server/api/general.js#L182-L190